src/commands/compile-go.yml

```diff
@@ -8,6 +8,8 @@ steps:
   - run:
       name: Go compile
       command: |
+        go env -w GOPRIVATE=$GOPRIVATE
```

Comment:
what does this line do?

Reply:
It sets GOPRIVATE var. I add this line for more readability, to set this var explicitly